pkg/types/baremetal/validation/platform.go

```diff
@@ -462,6 +457,48 @@ func ValidatePlatform(p *baremetal.Platform, agentBasedInstallation bool, n *typ
 	return allErrs
 }
 
+// GetEnabledCapabilities returns a set of enabled ClusterVersionCapabilities.
+func GetEnabledCapabilities(capabilities *types.Capabilities) sets.Set[configv1.ClusterVersionCapability] {
```

Comment:
This is not specific to baremetal right? For the semantics of it, I'd prefer if it was in a more general place to be used by all platforms.

Reply:
What about moving this to `pkg/types/installconfig.go`?
```
func (c *InstallConfig) EnabledCapabilities() sets.Set[configv1.ClusterVersionCapability] {
    [...]
}